sample/HttpTrigger-Powershell/modules/Get-WorkerProcess.psm1

```diff
@@ -0,0 +1,5 @@
+﻿Function Get-WorkerProcess
+{

```

Comment:
We run these samples outside of IIS Express (using Console ScriptHost). If it's easy to modify your example so it works in that case as well it would be good.